test/helpers/index.js

```diff
@@ -12,7 +12,7 @@ exports.create = async function (t, ...args) {
   const core = new Hypercore(db, ...args)
   await core.ready()
 
-  t.teardown(() => core.close(), { order: 1 })
+  t.teardown(() => core.close().catch(safetyCatch), { order: 1 })
```

Comment:
Note: this is not related to this PR, and I can remove it. But I think we should either
- Catch the error (assuming we allow `core.close()` to throw)
- Fail the test if `core.close()` throws (assuming it is not allowed to throw)

Went for catching, since I think it's allowed to throw